app/src/main/res/layout/fragment_info.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+xmlns:tools="http://schemas.android.com/tools"
+android:layout_width="match_parent"
+android:layout_height="match_parent"
+tools:context="com.mobileproto.hireddit.hireddit.visuals.InfoFragment"
+android:background="@color/colorBackground"
+android:paddingTop="15dp"
+android:paddingLeft="15dp"
+android:paddingRight="15dp"
+android:paddingBottom="50dp">
+
```

Comment:
can you add a back button to return to the previous fragment? i know android phones have the built in back button, but after looking at other builtin android apps they seem to have back buttons somewhere (upperleft corner)